doc/reference/trd105-shared-allow-syscalls.md

```diff
@@ -0,0 +1,95 @@
+Shared Allow System Call
+========================================
+
+**TRD:** 105 <br/>
```

Comment:
Technically, we don't assign a number until it's in its final stages (to avoid fragmentation). So you can just call  this trd-shared-allow-syscalls.md.